src/main/java/org/spongepowered/common/conversation/SpongeQuestion.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.common.conversation;
+
+import org.spongepowered.api.command.args.CommandElement;
+import org.spongepowered.api.conversation.AnswerHandler;
+import org.spongepowered.api.conversation.Question;
+import org.spongepowered.api.text.Text;
+
+import javax.annotation.Nonnull;
+
+public class SpongeQuestion implements Question {
+
+    private final String id;
+    private final Text prompt;
+    private final AnswerHandler handler;
+    private final CommandElement arguments;
+
+    SpongeQuestion(String id, Text prompt, AnswerHandler handler, CommandElement arguments) {
+        this.id = id;
+        this.prompt = prompt;
+        this.handler = handler;
+        this.arguments = arguments;
+    }
+
+    @Nonnull
```

Comment:
This should be already handled by the `@NonnullByDefault` annotation in the package.